commit/merkleroot/query.go

```diff
@@ -45,7 +45,7 @@ func (p *Processor) Query(ctx context.Context, prevOutcome Outcome) (Query, erro
 	for _, sourceChainRange := range prevOutcome.RangesSelectedForReport {
 		onRampAddress, err := p.ccipReader.GetContractAddress(consts.ContractNameOnRamp, sourceChainRange.ChainSel)
 		if err != nil {
-			lggr.Warnf("get onRamp address for chain %v: %w", sourceChainRange.ChainSel, err)
+			lggr.Warnf("get onRamp address for chain %v: %s", sourceChainRange.ChainSel, err)
```

Comment:
Lets have `err` in the structured log so that the tools currently available can easily surface it.

```suggestion
			lggr.Warnw(fmt.Sprintf("get onRamp address for chain %v: %s", sourceChainRange.ChainSel, err)), "err", err, "sourceChainRange", sourceChainRange)
```